src/send-query.js

```diff
@@ -0,0 +1,12 @@
+import ajax from './ajax';
+
+export default function sendQuery(query, config) {
+  const url = `https://${config.domain}/api/graph`;
+  const headers = {
+    authorization: `Basic ${btoa(config.apiKey)}`,
+    'content-type': 'application/json'
+  };
+  const body = JSON.stringify({ query });
```

Comment:
`config` is validated by the config class. query validation would require query parsing, and building a query parser is a lot of code. I have tests around query generation, but I don't think I want to actually be validating queries client-side at any point. The server will return a meaningful message if it's broken.